e2e/jest-puppeteer.config.js

```diff
@@ -2,9 +2,17 @@
  * This file is required by jest-puppeteer preset.
  * https://github.com/smooth-code/jest-puppeteer#configure-puppeteer
  */
+import { PuppeteerNode } from 'puppeteer';
 
 const fp = require('lodash/fp');
-const puppeteer = require('puppeteer');
+
+// TODO what settings are correct here?
+const puppeteer = new PuppeteerNode({
+  isPuppeteerCore: undefined,
+  preferredRevision: undefined,
+  projectRoot: undefined
+});
+
```

Comment:
I wouldn't try to make this into a module so `import` works. Maybe instead:
```javascript
const puppeteer = new require('puppeteer').PuppeteerNode({
  isPuppeteerCore: undefined,
  preferredRevision: undefined,
  projectRoot: undefined
});
```
Your usage below indicates to me that this will work.

Reply:
I tried this in various combinations but I wasn't successful.  I am guessing it's due to trying to import Typescript here where it doesn't belong.